src/reducers/l10n.js

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import type { Reducer, L10nState } from 'firefox-profiler/types';
+import { ReactLocalization } from '@fluent/react';
+
+/** This reducer holds the state for whether the localization is fethced or not and provides it*/
+const l10nReducer: Reducer<L10nState> = (
```

Comment:
Why don't we use the same way we do for other reducers with `combineReducers`? We have an object here with two state fields. we can split the `fetching` and `localization` to their own reducers and then use the `combineReducers` to combine them. It should look something like this:
```js
const isL10nFetching: Reducer<boolean> = (state = false, action) => {
  switch (action.type) {
    case 'REQUEST_L10N':
      return true;
    case 'RECEIVE_L10N':
      return false;
    default:
      return state;
  }
};

const localization: Reducer<ReactLocalization> = (
  state = new ReactLocalization([]),
  action
) => {
  switch (action.type) {
    case 'RECEIVE_L10N':
      return action.localization;
    default:
      return state;
  }
};

const l10nReducer: Reducer<L10nState> = combineReducers({
  isL10nFetching,
  localization,
});

export default l10nReducer;

```

That way, we split the reducers of both fields. It's a lot clear when we need to figure out what is changing the state of a particular field.

Reply:
Yeah this seems great. I wasn't quite sure about the convention for writing the reducers. I have changed these. Thanks!